requirements-devel.txt

```diff
@@ -13,3 +13,4 @@
 #     -e git+git://github.com/mitsuhiko/jinja2.git#egg=Jinja2
 
 -e git+https://github.com/inveniosoftware/flask-resources.git#egg=flask-resources
+-e git+https://github.com/inveniosoftware/invenio-resources.git#egg=invenio-resources
```

Comment:
major: does not exist? isn't it invenio-records-resources now? (though github keeps the old link working)

minor: flask-resrouces is released right? https://pypi.org/project/flask-resources/ probably we still need unreleased content. Worth adding already to setup.py

Reply:
Yes, probably best to make it match up to the new name. I've also added flask-resources in setup.py and removed it here. :rocket:

Reply:
ah no we need another release of flask-resources. Let's wait to do that change then.